pkg/apis/workflow/v1alpha1/workflow_types.go

```diff
@@ -818,6 +825,8 @@ type WorkflowStatus struct {
 
 	// Outputs captures output values and artifact locations produced by the workflow via global outputs
 	Outputs *Outputs `json:"outputs,omitempty" protobuf:"bytes,8,opt,name=outputs"`
+
+	PDBResourceName string `json:"pdbResourceName,omitempty" protobuf:"bytes,11,opt,name=pdbResourceName"`
```

Comment:
you don't need this field - you can identify any PBD for the workflow using lablels